modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -50,79 +50,130 @@ body {
 
 // Buttons
 
-.btn-link:not(.dropdown-item) {
-	@include button-cp(
-		var(--transparent),
-		var(--transparent),
-		var(--primary),
-		var(--transparent),
-		var(--transparent),
-		var(--primary-d1)
-	);
-}
+.btn:not(.btn-link) {
+	position: relative;
+	z-index: 1;
+
+	&::after {
+		background-color: black;
+		border-radius: clay-enable-rounded($btn-border-radius);
+		bottom: math-sign($btn-border-width);
+		content: '';
+		left: math-sign($btn-border-width);
+		opacity: 0;
+		position: absolute;
+		right: math-sign($btn-border-width);
+		top: math-sign($btn-border-width);
+		transition: opacity 150ms ease-in-out;
+		will-change: opacity;
+		z-index: -1;
```

Comment:
I need the combination of `z-index` to avoid this bug

without z-index
<img width="103" alt="Screenshot 2020-08-27 at 11 14 54" src="https://user-images.githubusercontent.com/46778114/91426908-6efe8d00-e85d-11ea-85e0-2b3779ef675b.png">

with z-index
<img width="106" alt="Screenshot 2020-08-27 at 11 15 07" src="https://user-images.githubusercontent.com/46778114/91426910-6f972380-e85d-11ea-9d10-f85ee69b42ec.png">